README.md

```diff
@@ -25,9 +25,43 @@ Ones you start your koa application, it will add several service endpoints such
 The list of service endpoints and examples of responses is below:
 
 ### /health
+Performs health checks and returns the results:
 ```
-{"status":"UP"}
+{
+  "status": "UP",
+  "db": {
+    "status": "UP",
+    "freeConnections": 10
+  },
+  "redis": {
+    "status": "UP",
+    "usedMemory": "52m",
+    "uptimeDays": 16
+  }
+}
+```
+The statuses of the health checks are aggregated in the root-level `status` field. If at least one check yields `DOWN` status,
+the aggregated status will become `DOWN`. Health checks can be defined in actuator options:
+ ```js
+app.use(actuator({
```

Comment:
Does it make sense to change the client API slightly? 
1. separate `checks` and `options` params. 
2. `checks` param requires a flat structure of services

```javascript
// actuator signature
function actuator(checks = {}, options = {}) {...};


// client code
var checks = {
  db: await db.status(),
  redis: await redis.status(),
};

app.use(actuator(checks))



```